exec/java-exec/src/main/java/org/apache/drill/exec/store/schedule/BlockMapBuilder.java

```diff
@@ -246,12 +249,16 @@ public EndpointByteMap getEndpointByteMap(FileWork work) throws IOException {
         DrillbitEndpoint endpoint = getDrillBitEndpoint(host);
         if (endpoint != null) {
           endpointByteMap.add(endpoint, bytes);
-        } else {
-          logger.info("Failure finding Drillbit running on host {}.  Skipping affinity to that host.", host);
+        } else if (logger.isDebugEnabled()) {
```

Comment:
and `} else if (noDrillbitHosts != null && noDrillbitHosts.add(host)) { logger.debug(...); }`